specs/protocol/README.md

```diff
@@ -93,42 +106,80 @@ see [here](#signatures) for more details
+| `currencyCode` | string                            | Y        | ISO 3166 currency code string                          |
+| `min`          | [`DecimalString`](#decimalstring) | N        | Minimum amount of currency that the offer is valid for |
+| `max`          | [`DecimalString`](#decimalstring) | N        | Maximum amount of currency that the offer is valid for |
+| `methods`      | [`PayoutMethod[]`](#payoutmethod) | Y        | A list of payment methods to select from               |
+
+#### `PayinMethod`
+| field                    | data type                               | required | description                                                                                                                                         |
+| ------------------------ | --------------------------------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `kind`                   | string                                  | Y        | Type of payment method (i.e. `DEBIT_CARD`, `BITCOIN_ADDRESS`, `SQUARE_PAY`). considered to be a unique identifier                                   |
+| `name`                   | string                                  | N        | Payment Method name. Expected to be rendered on screen.                                                                                             |
+| `description`            | string                                  | N        | Blurb containing helpful information about the payment method. Expected to be rendered on screen. e.g. "segwit addresses only"                      |
+| `group`                  | string                                  | N        | value that can be used to group specific payment methods together e.g. Mobile Money vs. Direct Bank Deposit                                         |
+| `requiredPaymentDetails` | [JSON Schema](https://json-schema.org/) | N        | A JSON Schema containing the fields that need to be collected in the RFQ's selected payment methods in order to use this payment method.            |
+| `fee`                    | [`DecimalString`](#decimalstring)       | N        | Fee charged to use this payment method. absence of this field implies that there is no _additional_ fee associated to the respective payment method |
+| `min`                    | [`DecimalString`](#decimalstring)       | N        | minimum amount required to use this payment method.                                                                                                 |
+| `max`                    | [`DecimalString`](#decimalstring)       | N        | maximum amount allowed when using this payment method.                                                                                              |
+
+> [!IMPORTANT]
+> `kind` should be considered as a unique identifier that is used to _identify_ / _specify_ an individual payment method
```

Comment:
```suggestion
> `kind` should be a unique identifier of an individual payment method
```

Unclear to me what this would mean as an outsider coming in, I may think to myself, "_does this mean I should great a UUID?_" Perhaps we don't need this?

Reply:
+1 feels duplicative to the description in the table above